requirements.txt

```diff
@@ -0,0 +1,20 @@
+boto3==1.26.83
+botocore==1.29.83
+certifi @ file:///croot/certifi_1671487769961/work/certifi
+Cython==0.29.33
+jmespath==1.0.1
+joblib==1.2.0
+natsort==8.3.1
+numpy==1.21.6
+pandas==1.1.5
+pybedtools==0.9.0
+pysam==0.20.0
+python-dateutil==2.8.2
+pytz==2022.7.1
+s3transfer==0.6.0
+scikit-learn==1.0.2
+scipy==1.7.3
+six==1.16.0
+-e git+https://github.com/CenterForMedicalGeneticsGhent/svtk@52270d1ee5f50edd0d8f8c77ae11c8a788318b52#egg=svtk
```

Comment:
What's this do?

Reply:
I don't really know. This file is the result of `pip freeze > requirements.txt`

Reply:
requirements are defined in `setup.py` so I'm not sure this is needed

Reply:
I know but when running `pip install -e .` it gave me errors that pysam and cython weren't installed (which are both imported in setup.py). That the reason I added it there (just an easy step instead having to install those separately)